lte/gateway/c/core/oai/tasks/nas/esm/esm_information.c

```diff
@@ -183,8 +184,8 @@ status_code_e mme_app_handle_esm_information_t3489_expiry(
   struct ue_mm_context_s* ue_context_p = mme_app_get_ue_context_for_timer(
       mme_ue_s1ap_id, "Deactivate EPS Bearer Ctx Request T3489 Timer");
   if (ue_context_p == NULL) {
-    OAILOG_ERROR(
-        LOG_MME_APP,
+    OAILOG_ERROR_UE(
+        LOG_MME_APP, ue_context_p->emm_context._imsi64,
```

Comment:
if ue_context_p is null we cannot use it

Reply:
+1 on this, this will end up as a Segfault

Reply:
Resolved